Azure-Samples/azure-search-openai-demo
# [CI] Move cron trigger to dedicated nightly workflow for better CI behavior on forks

## Purpose

When GitHub sees a cron trigger in a workflow, it defaults to disabling that workflow entirely in forks. That's sad for our python-test.yaml, since it's still useful for forks to run that on PRs. I discussed this with a GitHubber at GitHub Universe today, and this is what he proposed: make a separate workflow for the cron trigger whose job is just to call the tests workflow.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* He tested in a fork: https://github.com/collinmcneese/azure-search-openai-demo
* We should see the test job still get run nightly, but tests still run in CI in forks
